host/exchange/protocol.go

```diff
@@ -71,7 +67,7 @@ func (p *ExchangeProtocol) Request(shareReq *motor.ShareRequest) error {
 	// }
 
 	payload := &types.Payload{
-		Owner: profile,
+		// Owner: profile,
```

Comment:
nit: commented out code

Reply:
Updated all of the nit in lastest push @mcjcloud